fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -268,6 +279,28 @@ void FixedLagSmoother::processQueue(fuse_core::Transaction& transaction)
   }
 }
 
+bool FixedLagSmoother::resetServiceCallback(std_srvs::Empty::Request& req, std_srvs::Empty::Response& res)
```

Comment:
Neither `req` nor `res` are used, so it might be good to make that clear here and in the header:

```
bool FixedLagSmoother::resetServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
```